### PR TITLE
Fixed various french translations

### PR DESCRIPTION
**Pull request by Kaldor37**
Fixed many french typographic mistakes
Fixed some translations
Added some TODOs for translations to be checked with their context

### CODE REVIEW

**Comment by gschueler** [comment]
There is a conflict with the french translation file, can you do a rebase on our current master?

**Comment by ls-initiatives** [comment]
I'll make a review of this proposal.

File: rundeckapp/grails-app/i18n/messages_fr_FR.properties [hunk 1]
```diff
@@ -618,24 +619,24 @@ Execution=Ex\u00e9cution
 Execution.plural=Ex\u00e9cutions
 Node=N\u0153ud
 Node.plural=N\u0153uds
-steps=00c9tape
+steps=\u00c9tape
 strategy=Strat\u00e9gie
 execute.on.the.server.node=Ex\u00e9cuter sur le n\u0153ud de serveur \:
-include.nodes.matching=Inclure les n\u0153uds correspondant \u00e0 \:
+include.nodes.matching=Inclure les n\u0153uds correspondants \u00e0 \:
```
Comment (ls-initiatives) [inline rundeckapp/grails-app/i18n/messages_fr_FR.properties]: This one should remain untouched : "correspondant"  (participe présent)

**Comment by gschueler** [comment]
@ls-initiatives I will make that change, thanks

**Pull request merged**